applications/openshift/master/file_permissions_kube_apiserver/tests/ocp4/4.13.yml

```diff
@@ -0,0 +1,4 @@
+---
+default_result:
+  master: FAIL
```

Comment:
From what I could test this rule is passing on OCP 4.13

Reply:
that's correct